runtime/src/main/java/dev/ebullient/micrometer/runtime/binder/vertx/VertxMeterBinderAdapter.java

```diff
@@ -19,21 +17,16 @@
 import io.vertx.core.spi.metrics.VertxMetrics;
 
 @Singleton
-public class VertxMeterBinderAdapter extends MetricsOptions implements MeterBinder, VertxMetricsFactory, VertxMetrics {
+public class VertxMeterBinderAdapter extends MetricsOptions implements VertxMetricsFactory, VertxMetrics {
```

Comment:
I think this needs to remain a MeterBinder for discovery? Either that, or something else w/in the Vert.x Binder needs to have the correct quarkus registry injected, and I'm missing that change

Reply:
It's not an actual binder, so figured it wasn't needed.

The registry is set here: https://github.com/ebullient/quarkus-micrometer-extension/pull/20/files#diff-1cddb4aa56dcbcc90d6e95b9bea4aa79R40

Reply:
Missed that. More obvious in revised version. ;)